cg/server/invoices/views.py

```diff
@@ -199,6 +199,9 @@ def modified_invoice(invoice_id, cost_center):
     if not logged_in():
         return redirect(url_for("admin.index"))
 
+    if cost_center not in ["KTH", "KI"]:
+        return redirect(url_for("admin.index"))
```

Comment:
Perhaps this should raise HTTPError with bad request (Status code 400) instead? Since anything but ki and kth is not a valid parameter